### PR TITLE
Setup Proxy on NextJS && Integrate Login, Logout API

### PR DESCRIPTION
**Pull request by vtanh1905**
Setup Proxy on NextJS && Integrate Login, Logout API

### CODE REVIEW

File: frontend/src/pages/books/[id].tsx [hunk 1]
```diff
@@ -9,12 +9,7 @@ export default function BookDetail() {
...
     <div className={styles.book_details}>
       <div className={styles.left_content}>
-        <Image
-          src='/images/sach/con_cho_nho_bia_thuong_bia_1.jpg'
-          width={400}
-          height={600}
-          alt=''
-        ></Image>
+        <img src='/images/sach/con_cho_nho_bia_thuong_bia_1.jpg' />
         <div>
```
Comment (vtanh1905) [inline frontend/src/pages/books/[id].tsx:13]: I will make it better in the future

File: frontend/src/pages/_app.tsx [hunk 2]
```diff
@@ -4,25 +4,30 @@ import Head from 'next/head'
...
-  const title = Component.title ? `Book Store | ${Component.title}` : 'Book Store'
-  
-  const Layout = Component.layout ? Component.layout  : LayoutDefault;
+  const { user, onLogout } = useAuth()
+
+  const title = Component.title
+    ? `Book Store | ${Component.title}`
+    : 'Book Store'
+
+  const Layout = Component.layout ? Component.layout : LayoutDefault
```
Comment (vtanh1905) [inline frontend/src/pages/_app.tsx:23]: I will make it cleaner in the future

Comment (hungxnguyen118) [reply frontend/src/pages/_app.tsx:23]: ok. thank you. I know this is just the simple example, but at the next time please upgrade better. 👍

**Review by hungxnguyen118 — APPROVED** [review]
LGTM

**Pull request merged**